contracts/gas-service/AxelarGasService.sol

```diff
@@ -22,6 +22,8 @@ contract AxelarGasService is InterchainGasEstimation, Upgradable, IAxelarGasServ
 
     address public immutable gasCollector;
 
+    function _setup(bytes calldata data) internal override {}
```

Comment:
this is needed if the gmp-sdk dependency is updated. Are you missing the change to package.json/lock.json?